test/suites/rt-upgrade-chopsticks-orchestrator/test-upgrade-chain.ts

```diff
@@ -100,6 +136,27 @@ describeSuite({
 
         it({
             id: "T4",
+            title: "Xcm migrations have runned, if any",
+            test: async ({ skip }) => {
+                if (runtimeName === "flashbox") {
+                    skip();
+                }
+                // Regardless of migrations, query should be in the latest version
+                // Unfortunately the api is not very friendly
+                const currentXcmVersion = runtimeName.includes("light")
+                    ? (await api.consts.xcmPallet.advertisedXcmVersion).toNumber()
+                    : (await api.consts.polkadotXcm.advertisedXcmVersion).toNumber();
```

Comment:
I can't find this in polkadot js, where is it?

Reply:
haha not deployed yet. but I cherry-picked. its the companion PR